src/Categories/Adjoint/Construction/Kleisli.agda

```diff
@@ -105,3 +107,54 @@ Free⊣Forgetful = record
     M.μ.η B ∘ M.η.η (F₀ B)                            ≈⟨ M.identityʳ ⟩
     C.id                                              ∎
   }
+
+
+module KleisliExtension where
```

Comment:
As you use `M.μ` a lot below, maybe open it locally to `KleisliExtension` to make things shorter?